recipes/libs/libcap.yaml

```diff
@@ -21,6 +21,7 @@ buildScript: |
         AR=$AR \
         CC=$CC \
         OBJCOPY=$OBJCOPY \
+        ${CROSS_COMPILE:+BUILD_CC=gcc} \
```

Comment:
This line is unnecessary because `BUILD_CC` is always set to `gcc` on line 20 above...

Reply:
hmm @Ferruck was faster than me. 2 month ago, i didn't realized that! so this is fixed already 👍